recipes/vulkan-validationlayers/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+import shutil
+
+required_conan_version = ">=1.32.0"
+
+
+class VulkanValidationLayersConan(ConanFile):
+    name = "vulkan-validationlayers"
+    description = "Khronos official Vulkan validation layers for Windows, Linux, Android, and MacOS."
+    license = "Apache-2.0"
+    topics = ("conan", "vulkan-validation-layers", "vulkan", "validation-layers")
+    homepage = "https://github.com/KhronosGroup/Vulkan-ValidationLayers"
```

Comment:
Small remark before collecting the final approvals, how about naming this recipe `vulkan-validation-layers`?
The 'L' in of `Layers` in `ValidationLayers` is a capital.

Reply:
there is no clear consensus here: https://repology.org/project/vulkan-validationlayers/versions

Reply:
Thanks for the link!
I won't push further.